config/metasrv.example.toml

```diff
@@ -32,6 +32,9 @@ store_key_prefix = ""
 ## - Using shared storage (e.g., s3).
 enable_region_failover = false
 
+## The datastore for meta server.
```

Comment:
Do we need to list alternative options for `backend`?

Reply:
pg is not ready now, I think we can add it when it's available.